Add support for headless Firefox

korkeala opened the pull request:
The new version of https://github.com/karma-runner/karma-firefox-launcher (1.1.0) added support for launching firefox in headless mode. This pull-requests adds `firefox-headless` karma-env to start FirefoxHeadless plugin with `lein doo`.

You need Firefox >= 56 and karma-firefox-launcher >= 1.1.0 for testing.

bensu commented:
Thanks for the work!

- Can you please add firefox-headless to the test suite?
- Do you see any easy way to add a check for the karma version and warn if the required version is there?

korkeala commented:
I added it to the test suite and the tests failed, though looking at the log I suspect something else to be the cause. 

I also added firefox-headless to CircleCI tests, but from what I've dug I don't think they have recent enough version of Firefox available (also the Chrome version is quite old, I wonder how the chrome-headless tests pass on CircleCI?).

> Do you see any easy way to add a check for the karma version and warn if the required version is there?

No, sorry.

miikka commented:
Thanks for the patch!
